app/poros/quote.rb

```diff
@@ -0,0 +1,9 @@
+class Quote
+  attr_reader :id, :author, :content, :category
```

Comment:
This looks good. I don't think that you need `:id` just `:author, :content, and :category`. Could you think of a way that you could get your tags out of the array into a comma separated string? :)